extensions/agent/src/mainController.ts

```diff
@@ -42,6 +43,9 @@ export class MainController {
 	private proxyDialog: ProxyDialog;
 	private notebookDialog: NotebookDialog;
 	private notebookTemplateMap = new Map<string, TemplateMapObject>();
+
```

Comment:
nit: But it'd be great to undo these space changes to keep things clean. I usually accomplish this by opening the raw view of the file in github and copy-pasting that into my editor.